src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -74,7 +70,7 @@ public void onCommand(Member sender, CommandContext context) {
         builder.addField(context.i18n("commands.userinfo.user_created"), FormatUtils.formatDateTime(userForInfo.getCreationTime()), true);
         builder.addField(context.i18n("commands.userinfo.user_id"), userForInfo.getId(), true);
         builder.addField(context.i18n("commands.userinfo.user_mutual_servers"), String.valueOf(userForInfo.getMutualGuilds().size()), true);
-
+        pageList.add(new PageObjects.EmbedPage(builder, false));
```

Comment:
wat

Reply:
I'm really not sure what you want me to do with this

Reply:
I'm just wondering why it was moved